tests/eosio.system_tests.cpp

```diff
@@ -4862,8 +4860,8 @@ BOOST_FIXTURE_TEST_CASE( b1_vesting, eosio_system_tester ) try {
 
    const name b1{ N(b1) };
 
-   issue( alice, core_sym::from_string("20000.0000"), config::system_account_name );
-   issue( bob,   core_sym::from_string("20000.0000"), config::system_account_name );
+   issue_and_transfer( alice, core_sym::from_string("10000.0000"), config::system_account_name );
```

Comment:
Revert back to original amount. The change in definition of `issue_and_transfer` should resolve duplicate transaction errors.

Reply:
Done.